src/features/components/organisms/navigation/nav-sections-container/section-column.tsx

```diff
@@ -35,18 +36,22 @@ const NavSectionColumn = ({ item }: NavSectionColumnProps) => {
                 gap="5x"
                 padding_block="4x"
                 padding_inline="5x"
-                className={styles.section_items_container}
+                className={dclsx({
+                    [styles.section_items_container]: !item.data.no_divider,
+                })}
             >
-                <Typography.Paragraph
-                    className={styles.section_items_title}
-                    textcolor="secondary"
-                    align="left"
-                    font_family="UBUNTU"
-                    mb="2x"
-                    visible={!item.data.title ? 'larger-than-tablet' : undefined}
-                >
-                    {item.data.title ? <Localize translate_text={item.data.title} /> : ''}
-                </Typography.Paragraph>
+                {!item.data.no_title && (
+                    <Typography.Paragraph
+                        className={styles.section_items_title}
+                        textcolor="secondary"
+                        align="left"
+                        font_family="UBUNTU"
+                        mb="2x"
+                        visible={!item.data.title ? 'larger-than-tablet' : undefined}
+                    >
+                        {item.data.title ? <Localize translate_text={item.data.title} /> : ''}
+                    </Typography.Paragraph>
```

Comment:
Can we use `&&`?
```suggestion
                        visible={!item.data.title && 'larger-than-tablet'}
                    >
                        {item.data.title && <Localize translate_text={item.data.title} />}
                    </Typography.Paragraph>
```

Reply:
hey man, no we can't we have to provide the empty string. if you check the trade dropdown in ROW, you'll see the empty string.